andela/noldor-ah-frontend
# #161290854 Sass Setup

#### What does this PR do?
Adds support for Sass files.

#### Description of task to be completed
- add support for Sass files
- gitignore the dist folder

#### How should this be manually tested
- Create a file with a `.scss` extension and add some sample styling
- Import the Sass file in a component and render it
- You should notice that the app will render successfully

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161290854

#### Screenshots
<img width="1077" alt="screen shot 2018-12-03 at 3 15 49 pm" src="https://user-images.githubusercontent.com/36507972/49378997-63c11080-f70e-11e8-993a-0b036908cc19.png">
<img width="1440" alt="screen shot 2018-12-03 at 3 15 38 pm" src="https://user-images.githubusercontent.com/36507972/49378998-63c11080-f70e-11e8-89f7-735d7a8bb272.png">
